packages/expo-constants/src/ExponentConstants.web.ts

```diff
@@ -15,6 +13,30 @@ declare var localStorage: Storage;
 
 const _sessionId = uuidv4();
 
+function getBrowserName(): string | undefined {
+  const agent = navigator.userAgent.toLowerCase();
+  switch (true) {
+    case agent.includes('edge'):
+      return 'Edge';
+    case agent.includes('edg'):
+      return 'Chromium Edge';
+    // @ts-ignore: window is not defined
```

Comment:
TS is set up to allow `window`. I just tested locally and VS Code didn't show any TS errors and `yarn tsc` succeeded. Remove these unnecessary `@ts-ignore` comments and fix your editor if it's complaining about `window` (but not `navigator`??)

Reply:
https://app.circleci.com/jobs/github/expo/expo/119077/parallel-runs/0/steps/0-114

```
PASS iOS src/__tests__/Constants-test.ts (5.8s)
PASS Android src/__tests__/Constants-test.ts
PASS Web src/__tests__/Constants-test.ts
FAIL Node src/__tests__/Constants-test.ts
  ● Test suite failed to run

    ReferenceError: window is not defined

      15 | 
      16 | function getBrowserName(): string | undefined {
    > 17 |   const agent = window.navigator.userAgent.toLowerCase();
         |                 ^
      18 |   switch (true) {
      19 |     case agent.includes('edge'):
      20 |       return 'Edge';

      at getBrowserName (src/ExponentConstants.web.ts:17:17)
      at Object.get deviceName [as deviceName] (src/ExponentConstants.web.ts:90:12)
      at _objectWithoutPropertiesLoose (../../node_modules/@babel/runtime/helpers/objectWithoutPropertiesLoose.js:10:25)
      at _objectWithoutProperties (../../node_modules/@babel/runtime/helpers/objectWithoutProperties.js:5:16)
      at Object.<anonymous> (src/Constants.ts:48:55)
      at Object.<anonymous> (src/__tests__/Constants-test.ts:1:1)

Test Suites: 1 failed, 3 passed, 4 total
Tests:       6 passed, 6 total
Snapshots:   0 total
Time:        15.132s
Ran all test suites in 4 projects.
error Command failed with exit code 1.
```

Reply:
Gotcha, that makes sense. It's a bit sloppy but maybe we should add '.node' to web linter rules.